src/main/java/selenium/page/LoginPage.java

```diff
@@ -0,0 +1,58 @@
+package selenium.page;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class LoginPage extends BasePage {
+
+    @FindBy(xpath = "//input[@name='login[username]']")
+    private WebElement emailField;
+    @FindBy(xpath = "//input[@name='login[password]']")
+    private WebElement passwordField;
+    @FindBy(xpath = "//*[@id='send2'][@class='action login primary']")
+    private WebElement signInButton;
+    @FindBy(xpath = "//*[@id='email-error']")
+    private WebElement errorMessageEmail;
+    @FindBy(xpath = "//*[@id='pass-error']")
+    private WebElement errorMessagePassword;
+    @FindBy(xpath = "//*[@data-bind='html: $parent.prepareMessageForHtml(message.text)']")
```

Comment:
you are using xpath with too long locators. Instead, you could use css here `[data-ui-id='message-error']` which is shorter. This is actual for many locators in your project